docs/Modules/common-modules/event-bus.md

```diff
@@ -9,4 +9,174 @@ parent: "Common Modules"
 grand_parent: "Modules"
 ---
 
-# eventBus
\ No newline at end of file
+# eventBus
+
+A general purpose event bus.
+
+It is used to communicate across a diagram instance.
```

Comment:
We need to clarify the terminology. We shouldn't use module, service, and component for the same thing. My proposal would be _module_ since that is what they actually are. This could then be changed to:

_It is used to communicate between modules of a diagram instance without coupling them._

Reply:
I like that :+1: